src/main/arm/createUiDefinition.json

```diff
@@ -63,7 +62,7 @@
                 "type": "Microsoft.Common.InfoBox",
                 "options": {
                     "icon": "Info",
-                    "text": "[concat(if(bool(basics('useTrial')), 'By accepting the IBM License Agreement you are accepting the Evaluation terms of the license. An evaluation deployment does not include WebSphere interim fixes (iFixes). The evaluation period runs for 60 days after which you must purchase entitlement to continue to use WebSphere.', 'Accept the IBM License Agreement to proceed with the deployment.'), ' <a href=\"https://ibm.biz/tWASBaseLicenseAzureVMs\" target=\"_blank\">IBM WebSphere Application Server</a>.')]"
+                    "text": "Accept the <a href='https://ibm.biz/tWASBaseLicenseAzureVMs' target='_blank'>IBM WebSphere Application Server License Agreement</a> to proceed with the deployment."
```

Comment:
License URL is for Base. It should use ND license URL.

Reply:
I wondered about that.  Can you please reply with the correct ibm.biz URL, @git4rk ?

Reply:
@edburns its https://ibm.biz/tWASNDLicenseAzureVMs

Reply:
Thanks, I will push another commit, @git4rk .